app/src/main/java/com/ncnf/map/ui/MapFragment.java

```diff
@@ -85,12 +79,12 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         materialSearchBar = getView().findViewById(R.id.searchBarMap);
 
         // Initialize Google Map with the callback onMapReady
-        mapView = (MapView) getView().findViewById(R.id.map);
+        mapView = getView().findViewById(R.id.map);
```

Comment:
you can user requierView() which guaranties no nullptrexcep

Reply:
Good call, I'll do that, thanks!